docs/development/index.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+***********************
+Contributing to Astropy
+***********************
+
+So you are interested in contributing to the Astropy Project?  Excellent! We
+love contributions! Astropy is open source, built on open source, and we'd
+love to have in our community.
+
+For the time being, please see our `CONTRIBUTING.md
+<https://github.com/astropy/astropy/blob/main/CONTRIBUTING.md>`_ document
+for an overview of ways you can contribute to Astropy.  The rest of this
+document is focused specifically on documentation for contributors and
+maintainers on how to work the Astropy repository for submitting
```

Comment:
"how to work the Astropy repository" seems a bit awkward and narrowly focused. Overall there is an end to end process for developing new code or doc updates which includes creating a development environment, working with git and GitHub, making local updates, running tests, and working through the GitHub and astropy PR process.

Reply:
I agree, and I struggled a bit on this wording.  I believe it encapsulates all of that (where "repository" is implying both on GitHub and working with the code locally).  But I can try to rethink this.  Any suggestions?  Or maybe, instead of trying to be overly concise, I can replace this with a list like you just wrote "creating a development environment, working with git and GitHub, making local updates, running tests, and working through the GitHub and astropy PR process"

Reply:
Replacing with the list would work. :smile: